luminance/src/pipeline.rs

```diff
@@ -280,22 +283,21 @@ impl<'a> Pipeline<'a> {
 
 /// An opaque type representing a bound texture in a `Builder`. You may want to pass such an object
 /// to a shader’s uniform’s update.
-pub struct BoundTexture<'a, L, D, P>
+pub struct BoundTexture<'a, L, D, S>
 where
   L: 'a + Layerable,
   D: 'a + Dimensionable,
-  P: 'a + Pixel,
-{
+  S: 'a + SamplerType, {
   unit: u32,
   binding_stack: &'a Rc<RefCell<BindingStack>>,
-  _t: PhantomData<&'a Texture<L, D, P>>,
+  _t: PhantomData<&'a (L, D, S)>,
```

Comment:
To answer your previous question, the `Texture` was there before I actually had a `Texture` as parameter to `BoundTexture` (it only had a single type parameter), but your change seems fine to me.